release-1.8/release_notes_draft.md

```diff
@@ -268,17 +268,18 @@ This section provides an overview of deprecated API versions, options, flags, an
 
 ## **Notable Features**
```

Comment:
Should we take out the `strong emphasis` markup from headings ?

Reply:
done